library/patient_tracker.inc.php

```diff
@@ -19,6 +19,7 @@
 *
 */
 require_once(dirname(__FILE__) . '/appointments.inc.php');
+require_once("../interface/globals.php");
 
```

Comment:
is this needed?

Reply:
Yes, the code in globals.php uses `ini_set` set the selected time zone in Globals as time zone for the script in which globals.php is included. So, wherever date/time functions, this should be included in script containing them.